src/js/angular/rest/mappers/aclmanagement-mapper.js

```diff
@@ -7,24 +7,28 @@ import {ACListModel} from "../../aclmanagement/model";
 
 /**
  * Maps the ACL rules response to ACListModel.
+ * If no scope provided, the rule is omitted.
+ *
  * @param {Object} response
  * @return {ACListModel}
  */
 export const mapAclRulesResponse = (response) => {
     const aclModel = new ACListModel();
     if (response && response.data) {
         response.data.forEach((rule) => {
-            aclModel.appendNewRule(
-                rule.scope,
-                rule.policy,
-                rule.role,
-                rule.operation,
-                rule.subject,
-                rule.predicate,
-                rule.object,
-                rule.context,
-                rule.plugin
-            );
+            if (rule.scope) {
```

Comment:
I'm not sure when this can actually happen, but isn't it better if we report such cases somehow?

Reply:
It turned out it couldn't happen. I removed the check